com.unity.shadergraph/Editor/GraphUI/GraphElements/Commands/ChangePreviewModeCommand.cs

```diff
@@ -2,15 +2,19 @@
 using UnityEditor.GraphToolsFoundation.Overdrive;
 using UnityEditor.ShaderGraph.GraphUI.DataModel;
 using UnityEditor.ShaderGraph.GraphUI.EditorCommon.CommandStateObserver;
+using UnityEditor.ShaderGraph.GraphUI.EditorCommon.Preview;
+using UnityEditor.ShaderGraph.GraphDelta;
 using UnityEngine.GraphToolsFoundation.CommandStateObserver;
 
 namespace UnityEditor.ShaderGraph.GraphUI.GraphElements.CommandDispatch
 {
+    using PreviewRenderMode = HeadlessPreviewManager.PreviewRenderMode;
+
     public class ChangePreviewModeCommand : ModelCommand<GraphDataNodeModel>
```

Comment:
Could probably tear this out cause we dont intend on shipping preview mode changing for the first sprint